classes/02class/exercises/c02-network01/andreazzabr/c02-network01.txt

```diff
@@ -0,0 +1,42 @@
+c02-network01.txt
+
+
+- 10.0.0.1/24
+IP in binary: `00001010.00000000.00000000.00000001`
+subnet in binary: `11111111.11111111.11111111.00000000`
+subnet in decimals: `255.255.255.0`
+private/public: `private`
+hosts: `251`
+
+
+- 192.168.0.1/16
+IP in binary: `11000000.10101000.00000000.00000001`
+subnet in binary: `11111111.11111111.00000000.00000000`
+subnet in decimals: `255.255.0.0`
+private/public: `private`
+hosts: `65531`
+
+
+- 249.165.166.135/30
+IP in binary: `11111001.10100101.10100110.10000111`
+subnet in binary: `11111111.11111111.11111111.11111100`
+subnet in decimals: `255.255.255.252`
+private/public: `public`
+hosts: `2` can you choose public IPs in AWS? i know in a private newtork it would be 0. because 1 is for the router 1 for dns? none left for host?  does this rule apply for public too?
```

Comment:
Good question. @caiocezart

Reply:
Nope. The dns Server doesn’t need to be on the same subnetwork.

Reply:
The exercise is not really about AWS itself. Just getting you to identify how many hosts are available for each network.
This network will have 4 ips less 2 (network and broadcast) and only two IP for hosting.

AWS related this can't be used for subnets as the minimum is /28

Reply:
Hey @caiocezart, in the exercise we are asking for `how many IP addresses for HOSTS are available on AWS subnets`. Should we change it for `how many IP addresses for HOSTS are available this network`?

Reply:
the exercise should be just available on normal subnets, not AWS. I will raise an issue to fix that